tests/test_welltest_dpds.py

```diff
@@ -24,42 +24,42 @@ def test_main(tmpdir):
     datafilepath = os.path.join(ECLDIR, ECLCASE)
 
     # defaults only
-    sys.argv = ["welltest_extract", datafilepath, "55_33-1"]
-    welltest_extract.main()
+    sys.argv = ["welltest_dpds", datafilepath, "55_33-1"]
```

Comment:
SIB prefers not to punch sys.argv, and use  `mocker` instead. See e.g. `test_summaryplot` (this mocker is not in use in all tests yet)

Reply:
And potentially cleaner test function by using pytest.mark.parametrize